### DISCUSSION TITLE
Reload not working

### DISCUSSION THREAD

**Discussion by smfelsher** [pallets/quart#187] [Q&A] [answered]
# Initial Problem
I'm following the tutorial on [YouTube](https://youtu.be/0aCHKGRJgUc) and Quart's hot-reload feature is not working for me. I didn't make it any further than 2:29 in the video.

My code is identical to the code in the video.
I used the following command to launch the app: `$ QUART_APP=todo_api:app quart run`
I verified that the app was working by visiting the URL in my browser; I saw `hello` in my browser.
I changed the response message in the `todo_api.py` file. I expected my terminal to re-display the start-up messages, but it behaved as if it did not detect the file change.
I refreshed my browser window, but I received the old message and not the new message.

Environment:

- Python version: 3.10.6
- Quart version: 0.18.0
- Pipenv version: 2022.8.24
- VSCode: 1.71.2
- OS: Linux x64 5.19.7-1-MANJARO

# Trying Hypercorn

I ran the following command: `$ hypercorn --reload todo_api:app`
Performed the same steps as described above and I saw the app reload in the terminal and I received the updated response.

It appears to me that the `reload` feature of the Quart development server does not work in my configuration.

I tried to open an issue (#185) but was told the video tutorial can't be fixed. _What's wrong with the tutorial video?_
_Why does this work with Hypercorn but not the Quart development server?_

**Comment by Vivanov98** [comment]
Can confirm I am getting the same behaviour on ubuntu 20.04.1

Running hypercorn directly seems to work though
`hypercorn app_module:app --reload`

**Reply by smfelsher** [reply]
@Vivanov98, thank you for the suggestion. This will get me through development. However, why doesn't it work with Quart?

**Reply by Vivanov98** [reply]
Had a look at hypercorn and it looks to me that when activated programmatically (as is done with quart run cmd) the flag `use_reloader` isn't used, but when activated via the hypercorn cmd, it does seem to be used.

I might investigate more tomorrow

**Comment by pgjones** [comment, marked as answer]
Fixed in 7634a5a56b757eb6c5602407114d587ace8f5590

**Reply by smfelsher** [reply]
Thanks for the speedy fix!
